microovn/Makefile

```diff
@@ -5,7 +5,13 @@ TOKENDISTLIB := lib/charms/microcluster_token_distributor/v0/token_distributor.p
 
 build: $(CHARMFILE)
 
-$(CHARMFILE): src/charm.py charmcraft.yaml metadata.yaml $(OVSDBLIB) $(TOKENDISTLIB)
+$(TOKENDISTLIB):
+	mkdir -p $(shell dirname $(TOKENDISTLIB))
+	wget "https://raw.githubusercontent.com/canonical/microcluster-token-distributor-operator/refs/heads/main/lib/charms/microcluster_token_distributor/v0/token_distributor.py" -O $(TOKENDISTLIB)
```

Comment:
Please make use of charmcraft tooling for this, e.g: https://documentation.ubuntu.com/charmcraft/stable/reference/commands/fetch-libs/

Also, separate commit for the charm library management changes.